app/styles/recap-situation.css

```diff
@@ -0,0 +1,66 @@
+.recap-situation {
+    border-top: none;
+    background-color: white;
+}
+
+.recap-situation h3 {
+    margin-top: 0;
+}
+
+.recap-situation .section-header {
+    margin-top: 21px;
```

Comment:
Use `em` 🙂
This way, not only will it scale better, it will also read properly like “add two blank lines before that title”.
This is valid for all `px` instructions.